packages/Babylonian-Core.package/BPTrace.class/instance/addTracedValue.forProbe..st

```diff
@@ -1,4 +1,6 @@
 private
 addTracedValue: aTraceValue forProbe: aProbeId
-
-	(self probes at: aProbeId ifAbsentPut: [OrderedCollection new]) add: aTraceValue.
\ No newline at end of file
+	
+	
+	(self probes at: aProbeId ifAbsentPut: [OrderedCollection new]) add: aTraceValue.
+	aTraceValue chronologicalPosition: self probesTraceValuesCount-1.
```

Comment:
Hm... would it be a lot of effort to replace probesTraceValuesCount by an instance variable that holds a simple integer? That code is kind of performance critical and I would like to avoid computing the list of all probes on every recording. :)

Reply:
No problem at all! I thought that way the consistency of counting up was higher and that we will at max sum up like 10 integers as collection save their count as `tally` but I can understand where you're coming from